libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -520,7 +520,8 @@ bool ContractCompiler::visit(FunctionDefinition const& _function)
 bool ContractCompiler::visit(InlineAssembly const& _inlineAssembly)
 {
 	ErrorList errors;
-	assembly::CodeGenerator codeGen(errors);
+	ErrorReporter errorReporter(errors);
```

Comment:
It's actually coming in from `AsmStack.h` by way of `AsmAnalysis.h`.

It'd probably be wise to give this its own include, though, because what I just described is brittle.